src/storage/src/hummock/sstable/forward_sstable_iterator.rs

```diff
@@ -104,6 +104,30 @@ impl SstableIterator {
 
         Ok(())
     }
+
+    // Only for compaction because it would not load block from sstablestore.
+    pub fn next_inner(&mut self) -> HummockResult<()> {
```

Comment:
We may rename it to `next_for_compactor`

Reply:
done.